### PR TITLE
Added the CONTRIBUTING file and updated the ProjectBestPractices

### LINKED ISSUE #43: Contributing Guidelines

### PR DESCRIPTION
**Pull request by 201flaviosilva**
### Description

- Added the CONTRIBUTING file;
- Updated the wiki/ProjectBestPractices with more code styles;

### Image/Video

N/A.

### Related Issue

**Closes #43**

### Does this PR introduce a breaking change? ⚠️

- [X] No
- [ ] Yes

### Tests

- [ ] I tested all game E2E;
- [ ] I just tested some scenes;
- [X] Not applicable.

### Check list

Before submitting this pull request, please ensure the following are completed:

- [X] [My commits follow the project conventions](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#commit-message-convections).
- [ ] My code follows the project's coding style and conventions.
- [X] I have updated the project documentation (if necessary).
- [ ] My code makes sense with the design.

## Additional Notes

Based:
- https://github.com/photonstorm/phaser/blob/master/.github/CONTRIBUTING.md
- https://gist.github.com/briandk/3d2e8b3ec8daf5a27a62
- https://chat.openai.com/ (obviously)

---

Thank you for your contribution to Olympus Gone Wild! Your effort is greatly appreciated, and it helps make this project better for everyone. 😄🎮🚀

### CODE REVIEW

File: wiki/ProjectBestPractices.md [hunk 1]
```diff
@@ -4,12 +4,59 @@ This page will guide you through setting up the project, provide information on
 
 - https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/index.html
 - https://docs.godotengine.org/en/stable/tutorials/best_practices/index.html
+- https://github.com/FlamingoFiestaStudio/OlympusGoneWild/blob/master/.github/CONTRIBUTING.md
 
 ## Code Styles
 
+### Naming Conventions
```
Comment (201flaviosilva) [inline wiki/ProjectBestPractices.md:11]: If you don't agree with something, please let me know

**Review by Gabriel-Sous-a — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Please follow our [Coding Guidelines/Code Styles](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#code-styles) to maintain consistency in the codebase. We recommend using Godot's scripting language best practices and adhering to the GDScript style guide.
+
+### Submitting a Pull Request
+
+1. Ensure your changes are on a feature branch, not the `main` branch;
+2. Commit your changes with clear and concise messages, [check here commit message convections](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#commit-message-convections);
+3. Push your branch to your GitHub fork: `git push origin 123-my-issue-title`, [check here for branch name convention](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#branch-naming-convention);
+4. Create a Pull Request (PR) from your branch to the `main` project repository;
+5. [Check here for pull request best practices](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#pull-requests);
```
Comment (Goncalo-FP) [inline .github/CONTRIBUTING.md:52]: Points 2 and 3 have their warning message after the description. At point 4 it should be the same.

**Review by LeoHabs — APPROVED** [review]
I think it hits all the points, great writing Mr Flavio, really liked it

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (201flaviosilva) [reply .github/CONTRIBUTING.md]: Nice caught

**Review by Goncalo-FP — APPROVED** [review]

**Pull request merged**